Cargo.toml

```diff
@@ -16,14 +16,16 @@ directories = "5.0.1"
 hex-literal = "0.4.1"
 rand = "0.8.5"
 sha2 = "0.10.8"
+tracing = { version = "^0.1.34", features = ["log"] }
+tracing-subscriber = "0.3.18"
 ureq = "2.9.6"
```

Comment:
better to move these to dev-deps.

Reply:
https://github.com/Layr-Labs/rust-kzg-bn254/pull/4/commits/10beec1701c1e85e6320e1fb9b2108e90ccd707b